icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -61,6 +60,242 @@
     @Deprecated
     protected final String subType;
 
+    /**
+     * Used by new draft APIs in ICU 68.
+     *
+     * @internal
+     */
+    private MeasureUnitImpl measureUnitImpl = null;
+
+    /**
+     * Enumeration for unit complexity. There are three levels:
+     * <p>
+     * - SINGLE: A single unit, optionally with a power and/or SI prefix. Examples: hectare,
+     * square-kilometer, kilojoule, one-per-second.
+     * - COMPOUND: A unit composed of the product of multiple single units. Examples:
+     * meter-per-second, kilowatt-hour, kilogram-meter-per-square-second.
+     * - MIXED: A unit composed of the sum of multiple single units. Examples: foot-and-inch,
+     * hour-and-minute-and-second, degree-and-arcminute-and-arcsecond.
+     * <p>
+     * The complexity determines which operations are available. For example, you cannot set the power
+     * or SI prefix of a compound unit.
+     *
+     * @draft ICU 68
```

Comment:
HERE AND ELSEWHERE:

You need to include `@provisional` boilerplate.  Search for previous examples.

Reply:
aha ... this means it is also for each variable of the enum too.

Reply:
Done